go.mod

```
@@ -13,7 +13,7 @@ require (
 	github.com/jedib0t/go-pretty/v6 v6.3.5
 	github.com/jfrog/build-info-go v1.4.0
 	github.com/jfrog/gofrog v1.2.0
-	github.com/jfrog/jfrog-client-go v1.19.0
+	github.com/jfrog/jfrog-client-go v1.19.1-0.20220810150424-4126c4b5aea7
```

Please uncomment the "replace" at the bottom of the file and add the version there
```suggestion
	github.com/jfrog/jfrog-client-go v1.19.0
```

---

Gotcha!